src/js/edu-benefits/1990e/config/form.js

```diff
@@ -23,6 +37,9 @@ const formConfig = {
   transformForSubmit: transform,
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
+  defaultDefinitions: {
```

Comment:
This is actually what prompted our chat yesterday (see https://github.com/department-of-veterans-affairs/vets-website/pull/5026/files?short_path=c48d4c5#diff-c48d4c5da3172b6c94db9bc6f8c70ad0), though I'm still lacking complete clarity.   We are going to make a 2nd pass at that doc to make it clear why/when/which defaultDefinitions are needed